adcm_aio_client/core/actions/_objects.py

```diff
@@ -66,6 +66,46 @@ class ActionsAccessor(NonPaginatedChildAccessor):
     class_type = Action
 
 
+class Upgrade(InteractiveChildObject):
```

Comment:
1. Where's `UpgradeNode` implementation ?
2. Can't it be inherited from `Action`? in most cases there are the same methods and API, I suppose

Reply:
added